src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs

```diff
@@ -198,7 +198,6 @@ public async Task DisposeCurrentStream()
             await _currentStream.DisposeAsync().ConfigureAwait(false);
             _openStreams.Remove((int)_currentStreamId);
             _currentStream = null;
-            _currentStreamId = -4;
```

Comment:
`_currentStreamId` is used in quite a few places. I'd expect there was a reason why it was set up to -4 here, e.g.:
https://github.com/dotnet/runtime/blob/9aaf6027cae3fe92e5d5903adaddfd1b0dbdb1fa/src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs#L266-L266

So are we sure this won't break anything else?